### PR TITLE
Adding files

### PR DESCRIPTION
**Pull request by Nishune**
*Added new challenges.html
*challenges.scss
*Added same menu in challenges.html as in index.html
*Fixed a temporary navigation link between index.html and challenges.html in header navigation.

### CODE REVIEW

**Comment by nilsishome** [comment]
It looks like a nice start!

File: index.html [hunk 1]
```diff
@@ -1,35 +1,35 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8">
-    <title>Hackerrr Escape Room - Challenge yourself online or on-site.</title>
+    <meta charset="utf-8" />
+    <title>Hacker Escape Room - Challenge yourself online or on-site.</title>
```
Comment (Nishune) [inline index.html:5]: Changed back to 1 "R", is that ok?

**Review by nilsishome — COMMENTED** [review]
This is my review.

File: index.html (hunk 1 above)
Comment (nilsishome) [reply index.html:5]: Makes sense.

**Pull request merged**